packages/rn-tester-e2e/scripts/run-e2e-tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Check if an argument is provided
+if [ $# -eq 0 ]; then
+  echo "Error: No platform provided."
+  exit 1
+fi
```

Comment:
can we check that only the valid arguments are passed? ex. right now if I pass "windows" it will work

Reply:
sure, done.